Core/GDCore/Project/ObjectConfiguration.cpp

```diff
@@ -20,7 +20,7 @@ namespace gd {
 
 ObjectConfiguration::~ObjectConfiguration() {}
 
-ObjectConfiguration::ObjectConfiguration() {}
+ObjectConfiguration::ObjectConfiguration(): is3DObject(false) {}
```

Comment:
can you explain briefly what this does?

Reply:
It's a initializer list in C++ (only works for constructor). It just give the default value to this variable. 

It's a bit old style, it should probably use " = false" directly at the declaration in the .h file (something that is supported only since "recent" C++ versions, so I kept the habit of doing this the old way).